Makefile

```diff
@@ -25,9 +25,13 @@ API_TYPES_GENERATED:=$(API_TYPES_DIR)/$(API_ZZ_GENERATED).go
 API_GO_HEADER_FILE:=pkg/apis/header.go.txt
 
 # Container image-related variables
+IMAGE_BUILD_CMD:= podman build
```

Comment:
Nit, remove space, please.  Also, I know the whole Makefile can do with a cleanup wrt. = vs. := but why add more inconsistencies into this section?